### PR TITLE
Fix support for runtime theme in the Visitor Code view

### PR DESCRIPTION
**Pull request by gugalo**
The previous solution didn't behave as expected. Tested it out and fixed issues.

**Jira issue:**
https://glia.atlassian.net/browse/MOB-1952

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/helper/Utils.java [hunk 1]
```diff
@@ -541,7 +543,7 @@ private static <T> T getConfiguration(
         return newConf != null ? newConf : oldConf;
     }
 
-    public static UiTheme getFullHybridTheme(UiTheme newTheme, UiTheme oldTheme) {
+    public static UiTheme getFullHybridTheme(@NotNull UiTheme newTheme, @NotNull UiTheme oldTheme) {
```
Comment (DavDo) [inline widgetssdk/src/main/java/com/glia/widgets/helper/Utils.java:546]: 👍, I like additions like this)

**Review by DavDo — APPROVED** [review]
LGTM

**Review by andrews-moc — APPROVED** [review]

**Comment by gugalo** [comment]
!merge

**Comment by gugalo** [comment]
!merge

**Pull request merged**